README.md

```diff
@@ -2,4 +2,43 @@
 
 Cell Painting and other pipelines from the Imaging Platform.
 
-**Note**: The `without_batchfile` version of a pipeline is identical to the version without the suffix, except that the `CreateBatchFiles` module is disabled. The `without_batchfile` version exists because [Distributed-CellProfiler](https://github.com/CellProfiler/Distributed-CellProfiler) currently does not support the use of batch files. When updating the pipeline, ensure that these two versions are always in sync. 
+## Guidelines for creating pipelines 
+
+1. For each pipeline (e.g. `analysis.cpppipe`), always create a `without_batchfile` version of a pipeline (e.g. `analysis_without_batchfile.cpppipe`) and store in the same directory. The `without_batchfile` version is identical to the version without the suffix, except that the `CreateBatchFiles` module is disabled. The `without_batchfile` version exists because [Distributed-CellProfiler](https://github.com/CellProfiler/Distributed-CellProfiler) does not support the use of batch files. When updating the pipeline, ensure that these two versions are always in sync. 
+
+2. The `LoadData` module should always be configured with these values in order that the load data CSV file can be specified on command line
```

Comment:
Not true, LoadData will always take the name of any CSV you pass it no matter what this is set to.

Reply:
Huh – I could bet it failed at some point.

I'll update the docs, indicating this is not required by CellProfiler, but is good practice so that it is easy to diff pipelines

Reply:
I decided to delete the guideline because the diffable argument can be made for pretty much any module